### DISCUSSION TITLE
How to best display featured topic on the homepage?

### DISCUSSION THREAD

**Discussion by OliveiraCs** [visual-framework/vf-core#1710] [Proposals]
**What is a featured topic?**
Featured topic is a block composed of an image, title, short description and a link or an actionable button, which is of strategic importance to the organisation and requires premium real-estate on the homepage. 
This is not a news story, it is usually a deliberate piece of content (report, landing page, survey, etc.) we want visitors to engage with.

**Use case:** 
It is relevant for both the EMBL-EBI homepage and  EMBL.org. 

              
![homepage v1](https://user-images.githubusercontent.com/93318924/139288992-7dbfda09-ecc2-47fc-a266-54ada85cc37f.png)

![EMBL org homepage](https://user-images.githubusercontent.com/93318924/139289244-88ac49f3-ba3a-44ce-aa1d-21f23fa52a17.png)

**Standard questions proposing component** 

- In a few words, what does this component look like?
Featured topic is a block composed of an image, title, short description and a link or an actionable button, which is of strategic importance to the organisation and requires premium real-estate on the homepage. 
This is not a news story, it is usually a deliberate piece of content (report, landing page, survey, etc.) we want visitors to engage with.

- In a few words, what does this component do?
The goal is to highlight content without a predefined section, with the ability to change it whenever required. It is meant to send people to another destination: landing page, document, survey, etc.

- Would a rebrand change the structure or layout of this component?
Not sure. 

- Can other websites use this component? Or is it only useful in your organisation or website?
Yes, currently it’s displayed both on EMBL-EBI and EMBL.org homepages. This component could also be used in other contexts and projects that required highlighting and displaying content in similar ways. Ideally, we should be able to use it beyond the homepage.

- Do you have any designs or concepts?
No, requires a new redesign.

- Have you done any user testing already?
No, the user testing will be conducted in phase 2 of the project (Feb to May 2022)

- How often do you expect to make use of this component?
Frequently, to change content as required. (eg. once every couple of months currently)

**Comment by nikiforosk** [comment]
A bit late to this but [the analytics are showing very little engagement with the featured topic](https://docs.google.com/spreadsheets/d/13-PHoMIGqdFfM2eVNYGSNONEIxvXxgpDCRjtEkC6iH0/edit#gid=1405789029&range=D31)*: Just 0.12% of the events on the home page (only 504 events out of a total of 429,291). In my opinion this doesn't justify the effort to come up with a new component for such a specific and not very popular use case.

I agree that using the hero in the middle of the page isn't a good idea as it is too loud visually and it disrupts the flow of the content of the page (this was also mentioned by the participants in the feedback sessions). It also introduces two h1s on the same page (which isn't an error strictly speaking but is unusual and [not recommended at least by WebAim](https://webaim.org/techniques/semanticstructure/#headings)).

I'd suggest to use [the traditional vf summary with decorative image](https://stable.visual-framework.dev/components/vf-summary/#vf-summary--has-decorative-image) as an alternative. As this will stretch horizontally, it will look different that the rest of the content on the home page which is arranged in columns. With the correct amount of spacing from the previous and the next section, it can create a subtle but distinctive effect.

*This document has limited viewing access.
